packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/DataVisAnnotationTask/DataVisAnnotationTask.js

```diff
@@ -61,7 +55,7 @@ class DataVisAnnotationTask extends React.Component {
     // TODO: Add in the status count along with the validations to not go over the max allowed
 
     return (
-      <StyledFieldset>
+      <Box>
```

Comment:
If `fieldset` is still the most appropriate element, this can be `<Box as='fieldset'>`

Reply:
This PR is removing `fieldset`, because the task area shouldn't be a form.